### PR TITLE
Added responsive and stylish Event Landing Page

### LINKED ISSUE #2323: Landing page for an Events website
**Describe the project you want to add with tech stack**
Landing page for "Thriving Cities"

**Screenshots (optional)**

image

image

image

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Landing page for an Events website #2323

Close #2323 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

https://github.com/pranjay-poddar/Dev-Geeks/assets/133862694/02c186e1-0d38-43b9-ad51-57a99bd33b19

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@PoulavBhowmick03 please use images or ny assets through link in your project not upload images, or anything in your PR, that link will redirect to that particular assets/images which store in any cloud.

Also please remove  `.DS_Store` files from your PR.

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 changes done

**Comment by PoulavBhowmick03** [comment]
image
image
@MohitGupta121 But I have already deleted all the images. Check the ss of my commit and the clone repo

File: Frontend-Projects/Events_Landing_Page/assets/css/.DS_Store
Comment (MohitGupta121) [inline Frontend-Projects/Events_Landing_Page/assets/css/.DS_Store:1]: Please remove these files from project they are still there use this command to delete all files at once in whole project `find . -name ".DS_Store" -type f -delete`

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@PoulavBhowmick03 PTAL

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 Done. Check

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@PoulavBhowmick03 if your changes only in Frontend-Projects/Events_Landing_Page then why you posting other folders in file changes like TinDog Website, Js-Game etc. Please remove unnecessary things from your PR. Thanks!

**Comment by PoulavBhowmick03** [comment]
I am sorry, I dont have any idea on what went wrong. I would close this PR, discard the commits and work on it later @MohitGupta121

**Pull request closed without merging**